string time
string place
string name
uint16 category_id
uint16 id
sensor_msgs/Image cropped_image
sensor_msgs/Image image_rgb
float64[] bbox
float64[] segm
float64 score


